lib/time/time.Time.js

```diff
@@ -361,7 +361,13 @@
 	};
 
 	function pad( number, digits ) {
-		return ( 1e12 + Math.abs( number ) + '' ).slice( -digits );
+		number = '' + Math.abs( number );
```

Comment:
Type conversion via `'' + ...` is very common in JavaScript. I don't like calling constructors like they are functions.